apps/api/src/app/events/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -142,10 +142,24 @@ export class SendMessageChat extends SendMessageType {
       )
     )[0];
 
+    if (!integration) {
```

Comment:
I get the feeling it doesn't actually reach this point. So the error is still not set. Would you mind double checking? I'll also take a look

Reply:
I think it does if the subscriber have it in channels...
I will add a detail there as well :)

Reply:
@ainouzgali check my last commit, should fix the problem you found 😄